MAINTAINERS.md

```diff
@@ -1,10 +1,21 @@
-Maintainers of this repository with their focus areas:
+@brian-brazil is the main/default maintainer.
```

Comment:
How about making the list below clearer:

```suggestion
@brian-brazil is the main/default maintainer. Some parts of the codebase have other maintainers:
```

Reply:
I wasn't sure how this would end up rendering, it's still not great after applying this change.